### ISSUE TITLE
Arm crash

### ISSUE THREAD

**Issue by PowerDi** [Mohamed5341/opencv-image#5]
I install the plugins in arm platform using Remote-ssh. But the vscode crash.

**Comment by Mohamed5341** [comment]
Sorry for hearing that, could you please try to uninstall it and try vscode to check if it is working fine. Then try to install an older version.

**Comment by PowerDi** [comment]
No. 0.8.0 can not work. Although Vscode do not exit, the plugin seems to load failure.
